Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -254,6 +383,7 @@ def get_asset_id(params):
 
 
 def send_asset_vuln_request(asset_id, date_range):
+    """Function to get an assets vuln request."""
```

Comment:
Let's remove all these comments at the start of functions that were added, I don't feel like they add additional information @barchen1

Reply:
Hi @guykeller (cc: @barchen1 ), made the code change to fix the issue and pushed the new version of the Tenable_io.py file to the branch in my fork,

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed comments from code